### PR TITLE
Fixes Issue #33, Adding script to install all packages at once

### PR DESCRIPTION
**Pull request by rithgan**
I have added the contribution.md file as you requested as well. Please review.

### CODE REVIEW

**Review by fave77 — CHANGES_REQUESTED** [review]
@rithgan No need of adding a seperate contribution file, it already exists, just modify the portion of it as I told you

You can find it inside `.github` folder

File: .github/CONTRIBUTING/CONTRIBUTING.md [hunk 1]
```diff
@@ -23,6 +23,9 @@ $ git clone https://github.com/<YOUR-GITHUB-USERNAME>/pbchess.git
 # navigate to the project's directory
 $ cd pbchess
 
+Install the dependencies for both client and server
```
Comment (fave77) [inline .github/CONTRIBUTING/CONTRIBUTING.md]: Add the line using a comment

**Review by fave77 — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING/CONTRIBUTING.md (hunk 1 above)
Comment (fave77) [reply .github/CONTRIBUTING/CONTRIBUTING.md]: Also don't capitalise the first letter for consistency

Comment (rithgan) [reply .github/CONTRIBUTING/CONTRIBUTING.md]: Please review now

**Review by fave77 — APPROVED** [review]
@rithgan Looks good to me 👍🏻 
Thanks for the insight, it will help installing packages seamlessly for many contributors

**Pull request merged**